### PR TITLE
Add policy operator isIn

### PR DESCRIPTION
**Pull request by anatheka**
This PR implements the operators:
- isIn
- allIn

### CODE REVIEW

File: policies/operators.rego [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package clouditor
+import future.keywords.every # includes also in
 
+# TODO(anatheka): https://play.openpolicyagent.org/p/iNn3sOVG0Q
```
Comment (oxisto) [inline policies/operators.rego:4]: Can this todo be removed?

**Review by oxisto — COMMENTED** [review]
Looks interesting :)

File: policies/operators.rego (hunk 1 above)
Comment (anatheka) [reply policies/operators.rego:4]: Ups, yes. That was just for testing.

Comment (anatheka) [inline policies/operators.rego:4]:
```suggestion
```

**Review by oxisto — APPROVED** [review]

**Pull request merged**